mcp/run_tools.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: default
+title: Run MCP tools
+nav_order: 2
+parent: MCP - Query data interactively with an AI agent
+---
+
+{:.no_toc}
+# Run MCP tools
+
+This page shows you how to run a local agent and connect to a server running locally or remotely.
+
+* TOC
+{:toc}
+
+
+We provide specific instructions for the following agents:
+
+- [Gemini CLI](https://github.com/google-gemini/gemini-cli) 
+   - Can be used for datacommons.org or a Custom Data Commons instance
+   - Requires minimal setup 
+
+   See [Use Gemini CLI](#use-gemini-cli) for this option.
+- A sample basic agent based on the Google [Agent Development Kit](https://google.github.io/adk-docs/) and [Gemini Flash 2.5](https://deepmind.google/models/gemini/flash/) 
+   - Best for interacting with a Web UI
+   - Can be used for datacommons.org or a Custom Data Commons instance
+   - Can be customized to run other LLMs
+   - Requires some additional setup
+
+   See [Use the sample agent](#use-the-sample-agent) for this option.
+
+For an end-to-end tutorial using a server and agent over HTTP in the cloud, see the sample Data Commons Colab notebook, [Try Data Commons MCP Tools with a Custom Agent](https://github.com/datacommonsorg/agent-toolkit/blob/main/notebooks/datacommons_mcp_tools_with_custom_agent.ipynb).
+
+For other clients/agents, see the relevant documentation; you should be able to reuse the commands and arguments detailed below.
+
+## Prerequisites
+
+For all instances:
+
+- A (free) Data Commons API key. To obtain an API key, go to <https://apikeys.datacommons.org> and request a key for the `api.datacommons.org` domain.
+- For running the sample agent or the Colab notebook, a GCP project and a Google AI API key. For details on supported keys, see <https://google.github.io/adk-docs/get-started/quickstart/#set-up-the-model>.
+- For running the sample agent locally, or running the server locally in standalone mode, install `uv` for managing and installing Python packages; see the instructions at <https://docs.astral.sh/uv/getting-started/installation/>. 
+- For running the sample agent locally, install [Git](https://git-scm.com/).
+
+> **Important**: Additionally, for custom Data Commons instances:
```

Comment:
Suggestion: Rename `## Configure environment variables` to `## Configure environment`, then move this note underneath the `### Custom Data Commons` subheader so that the custom-specific instructions are together.

Reply:
I think having the correct version of custom DC falls under "configuring the environment", but up to you!